### ISSUE TITLE
Questions about the usage of deeds.

### ISSUE THREAD

**Issue by sakurawald** [Shinmera/deeds#5]
Hello, i am using the library `deeds`, it's well degisned, and almost fit my need.
However, i have some question that i still don't understand:

**Question 1: what is the difference between `message-event` and `payload-event`?**
Since there is no `type` for a `place` in common lisp, the variable has no type, and only the data has type.
So it seems that i can use the `message-event` to delivery `any type of data` that takes 1 slot.
![image](https://github.com/user-attachments/assets/bf54fcf4-5b4a-4f77-a946-fb791a0baae5)

**Question 2: Is there a convenient way to delivery `keyword arguments` conveninently?**
In other words, i would like something like
```
(do-issue person-add-event :name "alice" :age 12)
```

Or i have to `pack the keyword arguments` into `one structure` (maybe `alist` or `plist`), and pass this structure as `:payload` instead.

**Comment by Shinmera** [comment]
The documentation for `message-event` says:

> A simple event merely used to deliver message strings.

Whereas a `payload-event` can deliver any type of object.

Typically you'll define event subclasses that define what values are valid and potentially also perform some kind of validation on them. If you want an unstructured approach, you could indeed use a `payload-event` and just pass a plist as a payload or whatever.

**Comment by sakurawald** [comment]
> The documentation for `message-event` says:
> 
> > A simple event merely used to deliver message strings.
> 
> Whereas a `payload-event` can deliver any type of object.
> 
> Typically you'll define event subclasses that define what values are valid and potentially also perform some kind of validation on them. If you want an unstructured approach, you could indeed use a `payload-event` and just pass a plist as a payload or whatever.

Thank you~
